bundles/specmate-cdo-server/src/com/specmate/cdoserver/config/SpecmateCDOServerConfig.java

```diff
@@ -0,0 +1,63 @@
+package com.specmate.cdoserver.config;
+
+import java.util.Dictionary;
+import java.util.Hashtable;
+
+import org.osgi.service.cm.ConfigurationAdmin;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.log.LogService;
+
+import com.specmate.common.OSGiUtil;
+import com.specmate.common.SpecmateException;
+import com.specmate.config.api.IConfigService;
+
+@Component(immediate = true)
+public class SpecmateCDOServerConfig {
+
+	public static final String PID = "com.specmate.cdoserver";
+
+	public static final String KEY_SERVER_PORT = "cdo.serverPort";
+	public static final String KEY_REPOSITORY_NAME = "cdo.repositoryName";
+
+	private ConfigurationAdmin configurationAdmin;
+
+	private IConfigService configService;
+
+	private LogService logService;
+
+	private String serverPort;
+
+	private String specmateRepository;
+
+	@Activate
+	private void activate() throws SpecmateException {
+		this.serverPort = configService.getConfigurationProperty(KEY_SERVER_PORT);
+

```

Comment:
Read here also the KEY_REPOSITORY_NAME and remove that configuration option from the DBProviders.